.github/workflows/preview-env-deploy.yml

```diff
@@ -28,9 +28,28 @@ jobs:
             secret/data/products/camunda-docs/ci/preview-environment PREVIEW_ENV_BUCKET_NAME;
             secret/data/products/camunda-docs/ci/preview-environment PREVIEW_ENV_GCLOUD_SA_KEY;
 
+      - name: Shorten commit SHA
+        run: echo "short_sha=$(echo ${{ github.sha }} | cut -c1-8)" >> $GITHUB_ENV
+
+      - name: Find previous deployment or tear-down comment
+        uses: peter-evans/find-comment@v3
+        id: find-comment
+        with:
+          issue-number: ${{ github.event.number }}
+          body-includes: <!-- preview-env -->
+
       - name: Install Dependencies
         run: npm ci
 
+      - name: Upsert comment with build status
+        uses: peter-evans/create-or-update-comment@v4
```

Comment:
this job needs to be moved before the `npm ci` step (this is where the build actually starts, with dep install)